internal/commands/secrets/update_test.go

```diff
@@ -0,0 +1,173 @@
+package secrets
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/utils/test/assert"
+	"github.com/10gen/realm-cli/internal/utils/test/mock"
+)
+
+func TestUserUpdateHandler(t *testing.T) {
+	projectID := "projectID"
+	appID := "appID"
+	app := realm.App{
+		ID:          appID,
+		GroupID:     projectID,
+		ClientAppID: "test-abcde",
+		Name:        "test",
+	}
+
+	testLen := 7
+	secrets := make([]realm.Secret, testLen)
+	for i := 0; i < testLen; i++ {
+		secrets[i] = realm.Secret{
+			ID:   fmt.Sprintf("secretID%d", i),
+			Name: fmt.Sprintf("secretName%d", i),
+		}
+	}
+	successOutput := func(t *testing.T, output string) {
+		assert.Equal(t, "01:23:45 UTC INFO  Successfully updated secret\n", output)
+	}
+	for _, tc := range []struct {
+		description string
+		testSecret  string
+		testName    string
+		testValue   string
+		expectedErr error
+		testOutput  func(*testing.T, string)
+	}{
+		{
+			description: "should return a successful message for a successful update for name and value",
```

Comment:
these test cases are not really testing all that much stuff different from themselves... this whole test table can be just one "happy path" test func

the one thing to add would be to make sure we're capturing and asserting on the args that are passed into `realmClient.UpdateSecretFn`

Reply:
edit: after looking closer, i see what you were getting at with these tests, and i agree with you they deserve to exist as such (nice stuff!).  going to leave these in here, but hopefully in the future our commands actually do something with this knowledge (either require --name or --value, or print back a message indicating specifically what was performed ... out of scope for this ticket though)